share/functions/pushd.fish

```diff
@@ -76,6 +76,6 @@ function pushd --description 'Push directory to stack'
     end
 
     # argv[1] is a directory
-    set -g -p dirstack $PWD
-    cd $argv[1]
+    cd $argv[1];
```

Comment:
Semicolon is unnecessary if you use a linebreak. `fish_indent` would remove it.

Reply:
Ah, understood.